### PR TITLE
💄 fix #3 configurate global css

### LINKED ISSUE #3: Set up Project
acceptance criteria 

- [ ] configurate global css
- [ ] add all assets to public folder

### PR DESCRIPTION
**Pull request by RonjaMeinecke**
• configurate global.css
• add primary app colours
• add fonts

### CODE REVIEW

**Review by lauravikanis — APPROVED** [review]
LGTM 🚀

**Review by cboenke — APPROVED** [review]
LGTM 🚀

File: styles/globals.css [hunk 1]
```diff
@@ -1,9 +1,46 @@
+@font-face {
+  font-family: "BAMBI";
+  src: url("/public/fonts/X_BAMBI.TTF") format("ttf");
```
Comment (mewiemers) [inline styles/globals.css:3]: dont need `/public` to import fonts

**Review by mewiemers — COMMENTED** [review]
nicy code 🐇

File: styles/globals.css (hunk 1 above)
Comment (RonjaMeinecke) [reply styles/globals.css]: done! ☺️

**Pull request merged**